### ISSUE TITLE
Vanilla + Hiby R6 gen 3 + a particular Flac file always stops at 39/40 seconde

### ISSUE THREAD

**Issue by lionmarsupial** [vanilla-music/vanilla#1168]
- Vanilla Music Version (Settings -> About): 1.3.0
- Android Version: 12
- Phone vendor / model: Hiby R6 gen 3
- Application was downloaded from (use `[x]`):
  - [ ] Play Store
  - [x] F-Droid
  - [ ] Other (Nightly Build)
- Did this work in a previous version?
  - [ ] Yes
  - [x] No
  - [ ] NA / Don't know.

## Description

I installed Vanilla Music Player 1.3.0 on a Hiby R6 gen 3 and I have an issue with one particular music file.

Somewhere near 39 sec or 40 sec with this one file, the music just stops and can't be resumed.

I tested the same file with the same device Hiby R6 gen 3 with Vanilla Music Player 1.2.2, same issue.
I tested the same file with the same device Hiby R6 gen 3 with Vanilla Music Player 1.1.0, same issue.

I tested the same file with another player (VLC) on the same device (Hiby R6 gen 3), no issue, then file is played all allong.
I tested the same file with another android device (on a Lg V40 ThinQ) using a Vanilla Music Player 1.1.0, no issue, the file is played all allong.

Note : I also have another reproducible issue with Vanilla 1.3.0 with 2 other files. When the first one is played, the sound will be OFF during the playing of the second file. If played alone, the sound of the second file will be ON without issue.

I would need help to be able to identify the issue Vanilla Music Player meets on this particular device with these particular files.

Thanks a lot for help

## Steps to reproduce the issue

Playing the music from the beginning until the player will stop playing by itself at exactly 40 second.

**Comment by lionmarsupial** [comment]
Activating "Enable readahead" Option in "Miscellaneous features" didn't solve the issue.

**Comment by Ceremony64** [comment]
Is this a device specific issue or does this also occur on another device, like a regular android smartphone?

I am not familiar with the hiby, but is the behavior the same regardless which output option (speaker, headphone jack or bluetooth) is used?

You could also try reencoding it losslessy and see if that changes anything, even try other codecs, be it lossless wav or lossy mp3.

Also, do other players also have issue playing this file?

**Comment by lionmarsupial** [comment]
Only on this device, Vanilla issue, same file no error on other player on the same device.

**Comment by Ceremony64** [comment]
Well, you could just reencode the file with the same codec, frequency and bit depth (lossless) or use a different codec altogether to seewhether the issue is the decoding itself or related to the resulting bitstream or waveform. You openend another ticket ( #1170) about playback issues with mp3 files.nSo maybe the codec isn't the issue for either problem but something else entirely.

I am just suggesting some things to determine the underlying issue as I don't have neither the player nor the music file.

As for (debug) logs, I dunno if vanilla itself generates any, but maybe there is something in the logcat? 🤔

**Comment by lionmarsupial** [comment]
Yes, the other issue is weird and 100% reproducible as this one.
I will try this end of this day.
I don't know logcat, I will see also.
Thanks for your replies (it helps).

**Comment by lionmarsupial** [comment]
logcat doesn't show any message when the issue occurs (nothing).
I used ffmpeg to change the compression and the bitrate : No problem at all, the file is played without any issue.
